### PR TITLE
Prepare sim swap subscriptions in alpha version for the meta release

### PR DESCRIPTION
**Pull request by bigludo7**
#### What type of PR is this?

Add one of the following kinds:

* subproject management

NOTE: @fernandopradocabrillo  I prepare changelog part

#### Which issue(s) this PR fixes:

Fixes #

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

**Review by fernandopradocabrillo — APPROVED** [review]
LGTM

File: code/API_definitions/sim-swap-subscriptions.yaml [hunk 1]
```diff
@@ -64,12 +64,12 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: wip
+  version: 0.1-alpha.1
```
Comment (hdamker) [inline code/API_definitions/sim-swap-subscriptions.yaml:67]:
```suggestion
  version: 0.1.0-alpha.1

```

File: documentation/API_documentation/API-Readiness-Checklist_sim-swap-subscriptions-0.1.0.alpha1.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# API Readiness Checklist
+
+Checklist for sim-swap-subscriptions 0.1.0-alpha1 in FALL24
```
Comment (hdamker) [inline documentation/API_documentation/API-Readiness-Checklist_sim-swap-subscriptions-0.1.0.alpha1.md:3]:
```suggestion
Checklist for sim-swap-subscriptions 0.1.0-alpha.1 in FALL24

```

File: documentation/API_documentation/API-Readiness-Checklist_sim-swap-subscriptions-0.1.0.alpha1.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  Y   |  Y  |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |  Y   |  Y  |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |  Y   | contained in API definition |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |  Y   | Not Relevant (NR)   |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |  N   | NR |
+|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |  N   | NR    |
+|  9 | Test result statement                        |   O   |         O         |    O    |    M   |  N   | NR   |
+| 10 | API release numbering convention applied     |   M   |         M         |    M    |    M   |  Y   |  Y    |
+| 11 | Change log updated                           |   M   |         M         |    M    |    M   |  N   | to be provided |
```
Comment (hdamker) [inline documentation/API_documentation/API-Readiness-Checklist_sim-swap-subscriptions-0.1.0.alpha1.md:17]: will be contained in the repository changelog (already added within #127):

```suggestion
| 11 | Change log updated                           |   M   |         M         |    M    |    M   |  N   | [link](/CHANGELOG.md) |

```

**Review by fernandopradocabrillo — APPROVED** [review]

**Pull request merged**